src/common/com/intellij/plugins/haxe/lang/psi/impl/HaxeReferenceImpl.java

```diff
@@ -969,7 +985,7 @@ public String traceMsg(String message) {
     if (null != message) {
       msg.append(message);
     }
-    return HaxeDebugUtil.traceMessage(msg.toString(), 120);
+    return HaxeDebugUtil.traceMessage(msg.toString(), 2048);
```

Comment:
Yeah.  I was getting way too much data, so I limited it.  This is only in debug modes anyway, so we can dial this back later if we need to.